transformations/question_neural_paraphaser/README.md

```diff
@@ -0,0 +1,38 @@
+# Dialog Rephrasing transformation
```

Comment:
I think the transformation uses a text to text model trained on question-question pairs and hence I think it would not be right to call it dialog paraphrasing.